src/iuse_actor.cpp

```diff
@@ -3190,6 +3193,16 @@ int heal_actor::get_disinfected_level( const player &healer ) const
     return disinfectant_power;
 }
 
+int heal_actor::get_stopbleed_level( const player &healer ) const
+{
+    if( bleed > 0 ) {
+        /** @EFFECT_FIRSTAID increases healing item effects */
+        return bleed + healer.get_skill_level( skill_firstaid );
```

Comment:
This seems like a bit of a dramatic effect on bleeding - I think that the bandage quality should play a higher role, or the effect could decay as the skill reaches higher levels.

Reply:
Would `level / 2` be better?

Reply:
Yeah, seems better to me - I think something like 2 sqrt(level - x)  - a curve that levels off in some way - would be better than that though.